libs/httpproxy/proxy-camel-servlet/src/main/java/org/codice/proxy/http/HttpProxyCamelHttpTransportServlet.java

```diff
@@ -191,7 +191,7 @@ protected void doService(HttpServletRequest oldRequest, HttpServletResponse resp
         return;
       }
     } catch (IOException e) {
-      LOG.warn("Could not send error due to: ", e);
+      LOGGER.warn("Could not send error due to: ", e);
```

Comment:
@Lambeaux Had a good point on another ticket where he suggested that stack trace tracing like this should be done at debug level and not at higher level. We should keep the higher level log with only the exception toString().